jsk_fetch_robot/jsk_fetch.rosinstall.melodic

```diff
@@ -58,10 +60,12 @@
     local-name: jsk-ros-pkg/jsk_robot
     uri: https://github.com/knorth55/jsk_robot.git
     version: fetch15
+# Use https://github.com/knorth55/app_manager_utils master branch
+# after https://github.com/knorth55/app_manager_utils/pull/40 is merged
```

Comment:
https://github.com/knorth55/app_manager_utils/pull/40
it is already merged

Reply:
I rebased https://github.com/jsk-ros-pkg/jsk_robot/pull/1452, in which this rosinstall problem is resolved.